app/src/main/java/android/app/lotus/view/home/articles/ArticleListScreen.kt

```diff
@@ -57,38 +57,22 @@ fun ScrollableList(articleCategories: List<String>, accountType: String){
                     .padding(16.dp),
                 horizontalAlignment = Alignment.CenterHorizontally
             ){
-                val buttonColors = ButtonDefaults.buttonColors(Color.Green)
-               /* THIS IS FOR THE NAVIGATION LATER
-                if(accountType == "hr"){
-                    NavButton(
-                        text = category,
-                        navController = navController,
-                        route = Routes.
-                    )
-                }else if(accountType == "manager"){
-                    NavButton(
-                        text = category,
-                        navController = navController,
-                        route = Routes.
-                    )
-                }else if(accountType == "employee"){
-                    NavButton(
-                        text = category,
-                        navController = navController,
-                        route = Routes.
-                    )
-                }else{
-                    //ERROR
-                }
-
-                */
+                val buttonColors = ButtonDefaults.buttonColors(Color.Blue)
 
                 Button(
                     onClick = {
-                        //Just for visualy seeing. This is to be deleted later
+                        if(accountType == "hr"){
```

Comment:
You can leave this onClick empty - this is not how the code will look later anyways